subprojects/plugins/src/integTest/groovy/org/gradle/api/plugins/JvmTestSuitePluginIntegrationTest.groovy

```diff
@@ -158,9 +158,16 @@ class JvmTestSuitePluginIntegrationTest extends AbstractIntegrationSpec {
                 testData project
             }
 
+            // Extract the artifact view for cc compatibility by providing it as an explicit task input
+            def artifactView = testDataConfig.incoming.artifactView { view ->
+                                   view.componentFilter { it in ProjectComponentIdentifier }
```

Comment:
Do you think it might be worth adding a method to `artifactView` to encapsulate and describe this project filter?

Reply:
Or even to generate the whole artifact view?

Reply:
I'll defer to @ljacomet on this; I think there are a few other improvements we've talked about for this API, such as lazily resolving attribute values.